build.gradle

```diff
@@ -142,7 +142,7 @@ liquibase {
             driver "com.mysql.cj.jdbc.Driver"
             url "jdbc:mysql://localhost:3306/basicSample"
             username "root"
-            password "achhadata"
+            password "Ravikiran@123"
```

Comment:
I think this is not needed since we are not using the database. But if an error shows up when the password is not correct, please do let us know. We'll remove the unwanted code.

Reply:
ok